catalog-info.yml

```
@@ -0,0 +1,19 @@
+
+apiVersion: backstage.io/v1alpha1
+kind: Component
+metadata:
+  name: 'Credentials'
+  description: 'The Open edX Credentials Service, which supports course and program certificates.'
+  links:
+    - url: 'https://edx-credentials.readthedocs.io/'
+      title: 'Documentation'
+      icon: 'Article'
+    - url: 'https://github.com/openedx/credentials'
+      title: 'Source Code'
+      icon: 'Code'
+  annotations:
+    github.com/project-slug: openedx/Credentials
```

from [OEP-55/ADR-01](https://github.com/openedx/open-edx-proposals/blob/10abba68be201d42447fbf50dcfdd06a231e10a5/oeps/processes/oep-0055/decisions/0001-use-backstage-to-support-maintainers.rst):
```suggestion
  annotations:
    # (Optional) Annotation keys and values can be whatever you want.
    # We use it in Open edX repos to have a comma-separated list of GitHub user
    # names that might be interested in changes to the architecture of this
    # component.
    openedx.org/arch-interest-groups: ""
```